### PR TITLE
Deploy 0.17.0

### PR DESCRIPTION
**Pull request by CyberHoward**
Deployment of v0.17 and monorepo re-structuring.

### CODE REVIEW

File: applications/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["contracts/*"]
+members = ["contracts/apps/*", "packages/bundles/wyndex"]
```
Comment (Buckram123) [inline applications/Cargo.toml]: YESSS All on-chain apps together now 🔥. Don't forget to include "contracts/adapters/*"

File: scripts/wasm-all.sh [hunk 2]
```diff
@@ -4,60 +4,48 @@
 if [[ $(arch) == "arm64" ]]; then
 image="cosmwasm/rust-optimizer-arm64"
 workspace_image="cosmwasm/workspace-optimizer-arm64"
-abstract_image="abstractsdk/workspace-optimizer-arm64"
+abstract_image="abstractmoney/workspace-optimizer-arm64"
```
Comment (Buckram123) [inline scripts/wasm-all.sh:7]: I guess workspace image not used anymore?

**Review by Buckram123 — COMMENTED** [review]
Seems much better!

File: applications/Cargo.toml (hunk 1 above)
Comment (Buckram123) [reply applications/Cargo.toml]: And not sure about wyndex bundle

File: scripts/wasm-all.sh (hunk 2 above)
Comment (CyberHoward) [reply scripts/wasm-all.sh:7]: It's still the workspace image, it's just a fork that allows featured builds!

Comment (Buckram123) [reply scripts/wasm-all.sh:7]: I meant if we plan to use `abstract_image` over `workspace_image` why keep both :laughing:

**Pull request merged**